main/src/cgeo/geocaching/command/MoveToListCommand.java

```diff
@@ -33,6 +33,8 @@ public void execute() {
             @Override
             public void call(final Integer newListId) {
                 MoveToListCommand.this.newListId = newListId;
+                final String newListName = StoredList.getListById(newListId).getTitle();
+                MoveToListCommand.super.setProgressMessage(String.format(getContext().getString(R.string.command_move_caches_progress), newListName));
```

Comment:
Please use the variable arguments version of `getString()` instead of `String.format`.